File: mobile-app/CinemaBookingSolution/CinemaBookingCore/Data/Models/SeatCollectionModel.cs




using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CinemaBookingCore.Data.Entities;
using CinemaBookingCore.Data.Models;

namespace CinemaBookingCore.Data.Models
{
    public class SeatCollectionModel
    {
        public int ScheduleId { get; set; }
        public List<Ticket> TicketModels { get; set; }
        public Boolean isSuccesBookingTicket { get; set; }
    }
}

File: mobile-app/app/CinemaApplication/app/src/main/java/com/example/luulac/cinemaapplication/data/models/SeatCollectionModel.java
package com.example.luulac.cinemaapplication.data.models;

        import java.io.Serializable;
        import java.util.List;

public class SeatCollectionModel implements Serializable{
    private int scheduleId;
    private List<TicketModel> ticketModels;
    private boolean isSuccesBookingTicket;

    public SeatCollectionModel() {
        this.isSuccesBookingTicket = false;
    }

    public SeatCollectionModel(int scheduleId, List<TicketModel> ticketModels) {
        this.scheduleId = scheduleId;
        this.ticketModels = ticketModels;
        this.isSuccesBookingTicket = false;
    }

    public int getScheduleId() {
        return scheduleId;
    }

    public void setScheduleId(int scheduleId) {
        this.scheduleId = scheduleId;
    }

    public List<TicketModel> getTicketModels() {
        return ticketModels;
    }

    public void setTicketModels(List<TicketModel> ticketModel) {
        this.ticketModels = ticketModel;
    }

    public boolean isSuccesBookingTicket() {
        return isSuccesBookingTicket;
    }

    public void setSuccesBookingTicket(boolean succesBookingTicket) {
        isSuccesBookingTicket = succesBookingTicket;
    }
}
